src/opnsense/mvc/app/views/OPNsense/Proxy/index.volt

```diff
@@ -466,7 +466,6 @@
                             <td></td>
                             <td>
                                 <button data-action="add" type="button" class="btn btn-xs btn-default"><span class="fa fa-plus"></span></button>
-                                <button data-action="deleteSelected" type="button" class="btn btn-xs btn-default"><span class="fa fa-trash-o"></span></button>
```

Comment:
@fichtner this button did nothing so I removed it

Reply:
Ok if @AdSchellevis doesn’t want to keep it

Reply:
@fabianfrz @fichtner it can go, I guess someone copied it in there a long time ago....

Reply:
@AdSchellevis It did probably come from IDS because it is the only view with a list containing checkable checkboxes.